src/modules/nameSchema/assets/helper/numeral.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * Convert a number to a roman numeral.
+ * @param num - Number to convert.
+ * @returns Roman numeral.
+ * @see https://stackoverflow.com/a/41358305
+ */
+export const convertNumberToRoman = (num: number): string => {
+    const roman: Record<string, number> = {
+        M: 1000,
+        CM: 900,
+        D: 500,
+        CD: 400,
+        C: 100,
+        XC: 90,
+        L: 50,
+        XL: 40,
+        X: 10,
+        IX: 9,
+        V: 5,
+        IV: 4,
+        I: 1,
+    };
+    let str = '';
+    let rest = num;
+
+    for (const i of Object.keys(roman)) {
+        const q = Math.floor(rest / roman[i]);
+        rest -= q * roman[i];
+        str += i.repeat(q);
+    }
+
+    return str;
+};
+
+/**
+ * Convert a number to a string of letters.
+ * @param num - Number to convert.
+ * @returns String of letters.
+ */
+export const convertNumberToAlpha = (num: number): string => {
+    let str = '';
+    let rest = num;
+
+    do {
+        const q = rest % 26;
+        rest = Math.floor(rest / 26);
+        str = String.fromCharCode(65 + q) + str;
+    } while (rest > 0);
+
+    return str;
+};
+
+/**
+ * Convert a number to ICAO alphabet word.
```

Comment:
Do we get the german spelling alphabet as well? Eg. [Hamburg uses it](https://wiki.lstsim.de/FEZ_Hamburg/Fahrzeuge#Feuer-_und_Rettungswache_11_.28Innenstadt.29).

Reply:
❤️

Reply:
https://www.placetel.de/blog/buchstabieren-am-telefon-buchstaben-alphabet for future reference ;)

Reply:
@lithium040 already implemented in https://github.com/LSS-Manager/LSSM-V.4/pull/2872/commits/721065b7366ada166d5853af24b46e1a00407764 ✌🏻